build.sbt

```diff
@@ -4,5 +4,11 @@ version := "1.0"
 
 scalaVersion := "2.11.8"
 
-
-libraryDependencies += "org.scalatest" %% "scalatest" % "2.2.6" % "test"
\ No newline at end of file
+libraryDependencies ++= Seq(
+  "net.databinder.dispatch" %% "dispatch-core" % "0.11.2",
+  "org.json4s" %% "json4s-jackson" % "3.2.11",
+  "com.typesafe.akka" %% "akka-actor" % "2.4.0",
```

Comment:
Let's move version of akka as a variable. We've faced with issues with different Akka versions